src/common/debug.ml

```diff
@@ -169,7 +169,7 @@ let record_time, print_time =
 let stream_iter : ('a -> unit) -> 'a Stream.t -> unit =
   if not !do_print_time then Stream.iter else fun f strm ->
   let peek strm =
-    let open Stdlib in let r = ref None in
+    let r = ref None in
```

Comment:
@fblanqui I guess you used `open Stdlib` to avoid `Timed` references but `Timed` isn't opened there.